database_search_engine.py

```diff
@@ -32,6 +32,13 @@ def close(self): #отключение от БД
         if self.con:
             self.con.close()
 
+    def ensure_connection(self):
+        while not self.con:
+            self.connect()
+            if not self.con:
+            # wait for a second
+                time.sleep(1)
```

Comment:
Ты сам сказал так сделать, нет?
--
Envoyé de l'application Mail.Ru pour Android vendredi, 09 septembre 2016, 03:49PM +03:00 de Cadovvl < notifications@github.com> :

>In  database_search_engine.py :
>> @@ -32,6 +32,13 @@ def close(self): #отключение от БД
>          if self.con:
>              self.con.close()
>  
> +    def ensure_connection(self):
> +        while not self.con:
> +            self.connect()
> +            if not self.con:
> +            # wait for a second
> +                time.sleep(1)
>Магическая константа
>—
>You are receiving this because you authored the thread.
>Reply to this email directly,  view it on GitHub , or  mute the thread .